tests/tidb_duplicate_data/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
```

Comment:
In fact we need such a case more:
* lightning start to loading
* panic on some statements (inject failpoint)
* restart lightning to finish loading
* check everything is ok